modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/StepsTable.scala

```diff
@@ -1023,6 +1032,24 @@ object StepsTable extends Columns {
     }.getOrEmpty
   }
 
+  private val computeScrollBarWidth: CallbackTo[Double] = {
+    import org.querki.jquery.$
+
+    ref.get.map(_.getDOMNode.toHtml).asCallback.map{
+      _.flatten.flatMap { tableNode =>
+        // Table has a Grid inside, which is the one actually showing the scroll bar.
+        $(tableNode).find(".ReactVirtualized__Table__Grid").get(0).toOption.map {
```

Comment:
Maybe we can use `querySelector` and put it inside the `Option`
`Option(tableNode.querySelctor(".React....A))`?